### PR TITLE
Update 3d-models folder and FAQ

### PR DESCRIPTION
**Pull request by JulianGro**
This PR updates the docs inside the 3d-models folder and the FAQ.
I have added FIXMEs to the `pbr-materials-guide` as this will need more work.

Don't look at the branch name.

Preview available at https://data.moto9000.moe/sphinx/PR137/ (394a65f)

### CODE REVIEW

**Comment by digisomni** [comment]
Telling me to not look at the branch name makes me immediately look at the branch name!

File: docs/source/create/3d-models/get-model.rst [hunk 1]
```diff
@@ -68,10 +69,17 @@ We've listed the best practices for creating 3D models for Vircadia (and VR).
...
-| Textures   | Do not use PNG-48 or PNG-64, as neither are supported by High Fidelity.       |
+| Textures   | Do not use PNG-48 or PNG-64, as neither are supported by Vircadia.            |
 +------------+-------------------------------------------------------------------------------+
 | Textures   | When loaded in the engine, textures are automatically resized to a grid       |
-|            | of 128x128. Pick sizes which are multiples of 128.                            |
+|            | of 128x128. Pick sizes which are multiples of 128 to not waste any memory.    |
++------------+-------------------------------------------------------------------------------+
+| Textures   | While textures are automatically downscaled if they don't fit into the        |
+|            | texture memory, it makes sense to use a resolution as low as possible to      |
+|            | lower file sizes and lower the strain on low end machines from downscaling    |
```
Comment (ctrlaltdavid) [inline docs/source/create/3d-models/get-model.rst:79]:
```suggestion
|            | lower file sizes and lower the strain on low-end machines from downscaling    |
```

File: docs/source/create/3d-models/import-animations.rst [hunk 2]
```diff
@@ -15,29 +11,29 @@ Enrich your High Fidelity experience by having 3D models in your domain with ani
 Prerequisites
 ---------------------
 
-You need to be familiar with creating animations in 3D modeling tools such as Blender and Maya before importing an animation into High Fidelity.
+You need to be familiar with creating animations in 3D modelling tools such as Blender and Maya before importing an animation into Vircadia.
```
Comment (ctrlaltdavid) [inline docs/source/create/3d-models/import-animations.rst:14]:
```suggestion
You need to be familiar with creating animations in 3D modeling tools such as Blender and Maya before importing an animation into Vircadia.
```
US English has one "l". :o

File: docs/source/create/3d-models/import-animations.rst [hunk 3]
```diff
@@ -15,29 +11,29 @@ Enrich your High Fidelity experience by having 3D models in your domain with ani
...
 
 ---------------------------------
 Prepare a 3D Model Animation
 ---------------------------------
 
-Before you import an animation into High Fidelity, adjust some settings in the 3D modeling tool of your choice to ensure that it plays smoothly.
+Before you import an animation into Vircadia, adjust some settings in the 3D modelling tool of your choice to ensure that it plays smoothly.
```
Comment (ctrlaltdavid) [inline docs/source/create/3d-models/import-animations.rst:20]:
```suggestion
Before you import an animation into Vircadia, adjust some settings in the 3D modeling tool of your choice to ensure that it plays smoothly.
```

File: docs/source/create/3d-models/import-animations.rst [hunk 4]
```diff
@@ -15,29 +11,29 @@ Enrich your High Fidelity experience by having 3D models in your domain with ani
...
 
-1. Set the framerate to 30 fps for the best results (our recommendation).  
-2. Bake your animation channels, key frames, and inbetweens to ensure that High Fidelity reads everything. This is to ensure that your animation doesn't stop and start, but appears smooth and flows through each movement.
-3. Prepare to export the skeleton and frames that are being used in the animation. 
-4. Export your animation as an FBX file. 
-5. Upload this FBX file to a cloud server. Copy the URL. 
+1. We recommend to set the framerate to 30 fps as a good compromise between smoothness and file size.
```
Comment (ctrlaltdavid) [inline docs/source/create/3d-models/import-animations.rst:22]:
```suggestion
1. We recommend setting the framerate to 30 fps as a good compromise between smoothness and file size.
```

File: docs/source/create/3d-models/import-model.rst [hunk 5]
```diff
@@ -11,9 +11,9 @@ You can import a 3D model into Vircadia that is hosted online or on your domain'
 Import Models from the Cloud
 --------------------------------
 
-If you want your model available to users in other domains, we recommend uploading it to a cloud hosting service of your choice. 
+If you want your model available to users in other domains, we recommend uploading it to a cloud hosting service of your choice.
 
-1. Locate and copy the URL of the model you would like to import. The model should be a OBJ, glTF, or FBX format.
+1. Locate and copy the URL of the model you would like to import. The model should be an FBX, glTF, or OBJ file.
```
Comment (ctrlaltdavid) [inline docs/source/create/3d-models/import-model.rst:16]:
```suggestion
1. Locate and copy the URL of the model you would like to import. The model should be a glTF, FBX, or OBJ file.
```
To encourage glTF usage.

File: docs/source/faq.rst [hunk 6]
```diff
@@ -47,19 +41,53 @@ Domain Hosting
...
+
++-----------------------------------------------------------------------------------------------------+
+| .. _file-manager-help:                                                                              |
+|                                                                                                     |
+| How do I access the hidden folders that most Vircadia files are stored in?                          |
++=====================================================================================================+
+| On all currently supported operating systems the folders that Vircadia stores most of its data in   |
+| are hidden.                                                                                         |
+|                                                                                                     |
+| The easiest was to access them is to just copy the path and paste into your systems file manager.   |
```
Comment (ctrlaltdavid) [inline docs/source/faq.rst:82]:
```suggestion
| The easiest was to access them is to just copy the path and paste it into your systems file manager.|
```

File: docs/source/faq.rst [hunk 7]
```diff
@@ -47,19 +41,53 @@ Domain Hosting
...
+| Most (if not all) of them have a bar at the top.                                                    |
+|                                                                                                     |
+| .. image:: _images/file-manager-path.png                                                            |
+|                                                                                                     |
+| Even if it doesn't look like you can paste anything into the bar, still hover over it and try       |
+| pasting with a shortcut like Ctrl + V                                                               |
```
Comment (ctrlaltdavid) [inline docs/source/faq.rst:88]:
```suggestion
| pasting with a shortcut like Ctrl + V.                                                              |
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/create/3d-models/import-model.rst (hunk 5 above)
Comment (JulianGro) [reply docs/source/create/3d-models/import-model.rst:16]: I actually specifically put FBX there to encourage its usage. My understanding is that out FBX support is still better than our glTF support

File: docs/source/faq.rst (hunk 6 above)
Comment (JulianGro) [reply docs/source/faq.rst:82]:
```suggestion
| The easiest way to access them is to just copy the path and paste it into your systems file manager.|
```

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**